### PR TITLE
fix: add dark-theme to notes and textbook page

### PR DESCRIPTION
**Pull request by hinakhadim**
This PR adds dark-theme to notes and textbook page
### 1.Notes Page
image

### 2.Textbook Page
image

### 1.Notes Page
image
-----------------------------------------------------------------------------------------------
image
-----------------------------------------------------------------------------------------------
image
-----------------------------------------------------------------------------------------------
image
-----------------------------------------------------------------------------------------------
image

### 2. Textbook page
[The white page of pdf is rendered via pdfViewer/Canvas. It cannot be converted to dark in dark-mode.]
image

### CODE REVIEW

File: tutorindigo/templates/indigo/lms/static/sass/partials/lms/theme/_extras.scss [hunk 1]
```diff
@@ -298,33 +298,35 @@ nav.wrapper-preview-menu {
...
-          color: $light-dark;
-          &:hover, &:visited {
-            color: $dark;
-            border-color: transparent;
-          }
-          &.active {
-            color: $dark;
-            border-color: $primary;
-          }
+.wrapper-course-material[aria-label="Course Material"] {
```
Comment (DawoudSheraz) [inline tutorindigo/templates/indigo/lms/static/sass/partials/lms/theme/_extras.scss:301]: nit: any reason for moving it out of all body selectors?

**Review by DawoudSheraz — COMMENTED** [review]
please add a changelog entry

File: tutorindigo/templates/indigo/lms/static/sass/partials/lms/theme/_extras.scss (hunk 1 above)
Comment (hinakhadim) [reply tutorindigo/templates/indigo/lms/static/sass/partials/lms/theme/_extras.scss:301]: Yes, The `wrapper-course-material` is available on all pages (instructors, teams, wiki, static-page). so, if there is notes page and textbook page, we have to add `body.notes` for Notes page. For textbook page, there is no such class with body tag (it has `<body classname="indigo-dark-theme lang_en ltc" />` these classes). So, to apply on `textbook` page, I've made it generic and moved out of body classes. Also made it specific to tabs using the `aria-label` attribute.

File: changelog.d/20241209_134456_hina.khadim_teams_notes_page.md [hunk 2]
```diff
@@ -0,0 +1 @@
+- [BugFix] Update dark-theme styles for Notes and Textbook Page (by @hinakhadim)
```
Comment (Danyal-Faheem) [inline changelog.d/20241209_134456_hina.khadim_teams_notes_page.md:1]: nit: Change changelog attribute to `Bugfix`

**Review by Danyal-Faheem — APPROVED** [review]
LGTM! I tested these changes locally and they seem to be working as expected.

**Pull request merged**